CHANGES/883.bugfix

```diff
@@ -0,0 +1 @@
+Fix URL.build with port="" creating invalid URLs
```

Comment:
This should use RST markup for code, [past tense](https://yarl.aio-libs.org/en/latest/contributing/guidelines/#adding-change-notes-with-your-prs) and a byline with the `:user:` role in it:
```suggestion
Started treating an empty-string ``port=""`` passed into :py:meth:`~yarl.URL.build` as unset -- :user:`commonism`.

The logic has changed to create URL with no colon and port rendered in its string representation.
```